Overview.src.html

```diff
@@ -6243,7 +6243,10 @@ <h4 id="normalizing-a-sequence-of-keyframes">Normalizing a sequence of keyframes
 section></a> is performed before this operation is
 initiated.
 
-1.  If <var>frames</var> is a single {{Keyframe}} object, replaces
+1.  If <var>frames</var> is a {{SharedKeyframeList}} object, then
+    replace <var>frames</var> with the {{Keyframe}} or
+    sequence used to construct the {{SharedKeyframeList}}.
```

Comment:
Right, I think we need something else where we say what getFrames returns. The normalization procedure that we're changing here is explicitly called out as happening before we store the result. So if we pull out the set of keyframes from the SharedKeyframeList at this point then that's what we'll store so it won't be live or shared.